### ISSUE TITLE
Update Readme for Qiskit

### ISSUE THREAD

**Issue by born-2learn** [QPower-Research/QPowerAlgo#2]
Add details about Algorithms.

**Comment by diamondpowell** [comment]
I'd like to update the readme docs for Quantum Teleportation and Deutsch-Jozsa Algorithm

**Comment by born-2learn** [comment]
@DiamondPurvis Yes, you can send in a combined PR for the code and documentation of Quantum Teleportation and Deutsch-Jozsa Algorithms.
